lib/atlas/graph_deserializer.rb

```diff
@@ -0,0 +1,72 @@
+module Atlas
+  class GraphDeserializer
+    def self.build(graph_hash)
```

Comment:
* If `build` is the only public way to use this class (i.e. you don't intend for third-party code to ever run `new(...).build_graph`), you might want to add `private_class_method :new`: this will enforce that users of this class must use `call`.

  This also applies to AreaAttributesScaler and TimeCurveScaler.

* A little documentation describing what the `graph_hash` parameter is – and what the method returns – would be a nice addition.